src/hotspot/share/code/codeBlob.hpp

```diff
@@ -124,13 +124,17 @@ class CodeBlob {
 
   bool _caller_must_gc_arguments;
 
+  address _mutable_data;
+  int     _mutable_data_size;
+
```

Comment:
Should we add special CodeBlob subclass for nmethod to avoid increase of size for all blobs and stubs?

Reply:
I am not sure. All CodeBlobs with relocation info needs a mutable data. Let me know if you think it must be a separate subclass.